src/Action.py

```diff
@@ -0,0 +1,17 @@
+import chess_enums
+
+
+class Action:
+    orig: chess_enums.Space
```

Comment:
One thing I learned to make enums less verbose is "from chess_enums import Space". Then you can use Space instead of chess_enums.Space (same for the other enums).

Reply:
Also I didn't know you could specify the type of a variable like this in Python, I thought you had to do it through initialization. I may start doing it your way!

Reply:
Ooo I kept trying import chess_enums as Space, glad you had that figured out lol

Reply:
Fair warning, be careful with defining variables like that, you can specify the type but if you specify the class variable in that location it will be constant throughout all class instances as opposed to an instance variable